draft-demarco-oauth-status-assertions.md

```diff
@@ -631,7 +639,7 @@ an [SD-JWT.VC] is shown below.
 
 ### Credential Issuer Implementation Considerations
 
-When the Digital Credential is issued, the Credential Issuer SHOULD
+When the Digital Credential is issued, the Credential Issuer should
```

Comment:
I think reader will wonder why this isn't normative.

Reply:
because it is in a section for the considerations, I learned these kind of things by Mike :-)

or, still not learned but I remember this!

Reply:
You can have normative language in security considerations, but you are correct that normally the BCP 14 language is reserved for "ensuring interop".... this comes ups a surprising amount in IESG reviews.